tests/regression/44-trier_analyzer/41-strcpy.c

```diff
@@ -8,5 +8,9 @@ main () {
   strcpy (s, "Hello, strings!\n");
   strcpy (t, "Bye-bye!\n");
   y = strcpy (s, t);
+
+  assert(*y == 'B'); //TODO
+  char c = *y; //TODO
```

Comment:
What's the second TODO about? There's no assertion.

Reply:
Probably left over from some intermediate state or something.